README.md

```diff
@@ -64,14 +64,8 @@ Create a `database.yml` file on `config/` directory with your database configura
 ### Seed the database
 From the root of the app, run `bundle exec rails db:seed`. This will create some initial data to use while testing the app and developing new features, including setting up the default user.
 
-### Installing foreman
-Run the following command to install [foreman](https://github.com/ddollar/foreman).
-```bash
-gem install foreman
-```
-
 ### Start the app
-Run `bundle exec rails s` or `bin/start` (recommended since it runs webpacker in the background!) and browse to http://localhost:3000/
+Run `bundle exec rails server` or `bundle exec bin/start` (recommended since it runs webpacker in the background!) and browse to http://localhost:3000/
```

Comment:
Isn't `bundle exec rails s` short for `bundle exec rails server`?

Reply:
Yeap. We had discussed this before in an office hour. My position was/is that we generally want to stick with the non-abbreviated version in scripts/docs, since it's more transparent to newer devs (or devs newer to Rails) what we're trying to accomplish. I think there was agreement with that position, but it's also totes possible that the other people were just humoring me on my rant.